src/coreWebVitals.README.md

```diff
@@ -0,0 +1,72 @@
+# Core Web Vitals
+
+Reports on Core Web Vitals using Google’s [`web-vitals`] library, and send the
+metrics to an logging endpoint when the user leaves the page.
+
+By default, only 1% of page views will the data, but it’s possible to force this
+behaviour at initialisation or asynchronously.
+
+[`web-vitals`]: https://github.com/GoogleChrome/web-vitals
+
+## Usage
+
+```js
+import { initCoreWebVitals } from '@guardian/libs';
+
+const optionalCallback = () => {
+    // An optional callback which triggers when the user leaves the page
+    log('dotcom', 'CWV payload queued for async request')
+}
+
+initCoreWebVitals({
+    browserId,
+    pageViewId,
+    isDev, // will decide whether to use PROD or CODE enpoints
+}, optionalCallback)
+```
+
+### `init.forceSendMetrics`
+
+Allows to set at initialisation whether metrics should be sent
+for this page view.
+
+```ts
+const init = {
+    browserId: 'abc',
+    pageViewId: '123',
+    isDev: false,
+    forceSendMetrics: isInSeverSideTest(),
+}
+
+initCoreWebVitals(init)
+```
+
+### `forceSendMetrics`
```

Comment:
I find this confusing. Saying 'force' implies that it's an optional way to send the data in special circumstances and that simply calling  `initCoreWebVitals` is enough - is that true? Or do I _need_ to call this to have the metrics sent?

Reply:
+1. If they are part of the 1% group nothing is forced as they have been selected, so `init` will take place.

Reply:
Another good point. This is meant to bypass the sampling value. With your comments, I’m thinking of changing the init params to make this more explicit, and renaming this to `bypassSampling`.